negotiation/message/schema/contract-offer-message-schema.json

```diff
@@ -33,7 +33,7 @@
 					"type" : "string"
 				}
 			},
-			"required": [ "@context", "@type", "dspace:providerPid", "dspace:offer", "dspace:callbackAddress" ]
+			"required": [ "@context", "@type", "dspace:providerPid", "dspace:consumerPid", "dspace:offer", "dspace:callbackAddress" ]
```

Comment:
"dspace:consumerPid" is optional for a ContractOfferMessage, so please remove it from this array.